htdocs/core/class/html.formticket.class.php

```diff
@@ -1493,7 +1493,7 @@ public function showMessageForm($width = '40%')
 			$topic = "";
 			foreach ($formmail->lines_model as $line) {
 				if ($this->param['models_id'] == $line->id) {
-					$topic = $line->topic;
+					$topic = make_substitutions($line->topic, $this->substit);
```

Comment:
this->substit seems not initialized in function. What happen if var is null or unset ?